resources/views/response/faq/header.html.twig

```diff
@@ -0,0 +1,27 @@
+<header>
+    <div data-c-overlay="white(80)" style="background-image: url('/images/response-wall.jpg')">
+        <div data-c-container="large" data-c-padding="tb(3)" data-c-align="base(center) tl(left)">
+            <h1 data-c-heading="h1" data-c-color="black" data-c-margin="bottom(1)">{{ response_faq.page_title }}</h1>
+            <div data-c-grid="gutter(all, 1)">
+                <div data-c-grid-item="tl(2of3)" data-c-align="base(center) tl(left)">
+                    {# Breadcrumbs #}
+                    <div data-c-alignment="base(center) tl(left)" data-c-color="black">
+                        <nav aria-label="breadcrumb" data-c-breadcrumbs>
+                            <ol>
+                                <li>
+                                    <a href="/response" title="Return to the response homepage.">Home</a><i>/</i>
+                                </li>
+                                <li>
+                                    <span aria-current="page">FAQ</span><i>/</i>
+                                </li>
+                            </ol>
+                        </nav>
```

Comment:
You can replace this with the breadcrumbs view composer:  `{% include "common/breadcrumbs %}`